src/CMakeLists.txt

```diff
@@ -47,27 +46,40 @@ list(APPEND _packages
   Utilities
   )
 
+# These are directories that should be added but not made into targets
+list(APPEND _dir_packages
+  CXXTests)
+
+if(ENABLE_MPI)
+  # Add Distributed target
+  list(APPEND _target_packages Distributed)
+endif()
```

Comment:
In the Distributed folder we install a fakempi.py python script for non mpi builds. The failed ~mpi tests are because we aren't entering that directory anymore to install it.